### PR TITLE
Version 1.0 of nifty site manager (nsm)

### PR DESCRIPTION
**Pull request by n-ham**

### CODE REVIEW

File: org.nifty-site-manager.nsm.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        {
+            "name": "nsm",
+            "buildsystem": "simple",
+            "build-commands": [
+				"make",
+				"make install"
+            ],
+            "sources": [
+                {
+					"type": "archive",
```
Comment (barthalion) [inline org.nifty-site-manager.nsm.json]: You should use specific tag or commit here.

File: org.nifty-site-manager.nsm.desktop [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
```
Comment (barthalion) [inline org.nifty-site-manager.nsm.desktop]: See above.

File: org.nifty-site-manager.nsm.appdata.xml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (barthalion) [inline org.nifty-site-manager.nsm.appdata.xml]: Any reason not to ship it with the project? Other distributions also rely on appdata, desktop and icon files.

**Comment by barthalion** [comment]
bot, build org.nifty-site-manager.nsm

File: org.nifty-site-manager.nsm.appdata.xml (hunk 3 above)
Comment (barthalion) [reply org.nifty-site-manager.nsm.appdata.xml]: Also, you need to install all three in correct locations, that's why test build failed.

**Comment by LeandroStanger** [comment]
bot, build org.nifty-site-manager.nsm

**Pull request closed without merging**